content/docs/command-reference/fetch.md

```diff
@@ -45,17 +47,17 @@ Fetching could be useful when first checking out a <abbr>DVC project</abbr>,
 since files tracked by DVC should already exist in remote storage, but won't be
 in the project's <abbr>cache</abbr>. (Refer to `dvc remote` for more information
 on DVC remotes.) These necessary data or model files are listed as
-<abbr>dependencies</abbr> or <abbr>outputs</abbr> in a DVC-file (target
+<abbr>dependencies</abbr> or <abbr>outputs</abbr> in a `.dvc` file (target
 [stage](/doc/command-reference/run)) so they are required to
 [reproduce](/doc/tutorials/get-started/data-pipelines#reproduce) the
 corresponding [pipeline](/doc/command-reference/pipeline). (See
-[DVC-File Format](/doc/user-guide/dvc-file-format) for more information on
+[`.dvc` File Format](/doc/user-guide/dvc-file-format) for more information on
 dependencies and outputs.)
```

Comment:
Let's remove the whole last paragraph sentence here "(See ... outputs.)" please.